server/routes/incidents_routes.py

```
@@ -11,7 +11,7 @@
 from utils.auth.stateless_auth import get_org_id_from_request, set_rls_context
 from utils.log_sanitizer import hash_for_log, sanitize
 from chat.background.task import run_background_chat
-from typing import List, Dict, Any, Optional
+from typing import Any, Dict, List, Optional, Tuple
```

Ruff (0.16.3)

[warning] `typing.Dict` is deprecated, use `dict` instead

(UP035)